Documentation/reportgenerationqueries.md

```diff
@@ -11,7 +11,8 @@ When writing a [report](report.md) you can specify the query it will use by sett
 - `columns`: A list of columns that match the schema of the results of the query. The order of these columns must match the order of the columns returned by the SELECT statement. Columns have 3 fields, `name`, `type`, and `unit`. Each field is covered in more detail below.
   - `name`: This is the name of the column returned in the `SELECT` statement.
   - `type`: This is the [Hive][hive-types] column type. Currently due to implementation details, column types are expressed using hive types. In the future, this will likely be switched to using the Presto native types. This also has an effect that queries with columns containing complex types such as `maps` or `arrays` cannot be used by `Reports` or `ScheduledReports`.
-  - `unit`:
+  - `unit`: Unit refers to the measurement of the column.
```

Comment:
Normally I see the phrase "unit of measurement" rather than just "measurement", so maybe just adjust slightly like this?:
`Unit refers to the unit of measurement for the column.`

Reply:
all fixed in commit b04edc419b35fa20983f1ec9dfcfa082560962bf